packages/babel-plugin-syntax-export-default/README.md

```diff
@@ -0,0 +1,35 @@
+# @babel/plugin-syntax-export-default
+
+> Allow parsing of export default from.
```

Comment:
Nit: export default from should be in backticks so that it is formatted as code? (`export default from`)